### ISSUE TITLE
Asymmetry between sides.

### ISSUE THREAD

**Issue by aixiu** [iissnan/hexo-theme-next#1916]
### I agree and want to create new issue 

- [] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [] Yes, I was on [NexT Documentation Site](http://theme-next.iissnan.com/).
- [] And yes, I already searched for current [issues](https://github.com/iissnan/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Steps to reproduce the behavior 
1. N/A
2. N/A
3. N/A

* Link to demo site with this issue: N/A
* Link(s) to source code or any usefull link(s): N/A

### Hexo Information

```

```

### NexT Information

**NexT Version:**

```
[x] Latest Master branch.
[] Latest Release version.
[] Old version - 
```

**NexT Scheme:**

```
[] All schemes
[] Muse
[] Mist
[] Pisces
[x] Gemini
```

### Other Information 
![default](https://user-images.githubusercontent.com/8064012/31078536-18f13bf6-a7b5-11e7-9e56-8199fa4a2991.png)

**Comment by aixiu** [comment]
![default](https://user-images.githubusercontent.com/8064012/31214532-2b90e9ac-a9de-11e7-9ab7-3313e7b1d9c4.png)
Seems to be a block positioning is wrong. Sorry, I don't know how to fix it.

**Comment by ivan-nginx** [comment]
Take it easy, i already add «bug» label. It mean what bug was detected already and i know this bug. It will be fixed as soon as possible. This is not critical bug, i think.

**Comment by aixiu** [comment]
@ ivan-nginx Thank you, I'll be patient to fix it, goog job.

**Comment by ivan-nginx** [comment]
Fixed in `52710a5`.
